website/docs/guides/dbt-python-bigframes.md

```diff
@@ -0,0 +1,384 @@
+
+For example (`service account` credential)
+
+```yaml
+my_dbt_project_sa:
+  outputs:
+    dev:
+      dataproc_region: us-central1
+      dataset: jialuo_test_us
+      gcs_bucket: jialuo_bf_dbt_integration_dev1
+      job_execution_timeout_seconds: 300
+      job_retries: 1
+      keyfile: /usr/local/google/home/jialuo/dbt_t2/secrets/sa_key.json
+      location: US
+      method: service-account
+      priority: interactive
+      project: bigframes-dbt-integration-dev
+      threads: 1
+      type: bigquery
+  target: dev
+```
+
+OR ('Oauth' credential)
+
+```yaml
+my_dbt_project:
+  outputs:
+    dev:
+      dataproc_region: us-central1
+      dataset: jialuo_test_us
+      gcs_bucket: jialuo_bf_dbt_dev2
+      job_execution_timeout_seconds: 300
+      job_retries: 1
+      location: US
+      method: oauth
+      priority: interactive
+      project: bigframes-dev-perf
+      threads: 1
+      type: bigquery
+  target: dev
+```
+Please note, the bigframes submission method does **NOT** introduce any new configuration in profile.yml under $HOME/.dbt.
+
+5. Set Bigframes submission method
+
+Set bigframes submission method by either 
+
+(a) Project level configuration via dbt_project.yml, OR
+
+For example
+
+```python
+# Name your project! Project names should contain only lowercase characters
+# and underscores. A good package name should reflect your organization's
+# name or the intended use of these models
+name: 'my_dbt_project'
+version: '1.0.0'
+
+…
+
+# Configuring models
+# Full documentation: https://docs.getdbt.com/docs/configuring-models
+
+# In this example config, we tell dbt to build all models in the example/
+# directory as views. These settings can be overridden in the individual model
+# files using the `{{ config(...) }}` macro.
+models:
+  my_dbt_project:
+    submission_method: bigframes
+    # Config indicated by + and applies to all files under models/example/
+    example:
+      +materialized: view
+```
+
+(b) The Python code via dbt.config, which you will be creating later and can overwrite (a)
+
+For example
+
+```python
+def model(dbt, session):
+   # Will overwrite the submission method setup in dbt_project.yml
+   dbt.config(submission_method="bigframes")
+
+   bdf = dbt.ref("my_sql_model") #loading from prev step
+   <process bigquery dataframe >
+   return bdf
+```
+
+## Execute a python model
+
+1. **Add a new .py file on your dbt project**
+
+Now, you can add python models to your dbt project models folder.
+
+For example
+
+```python
+def model(dbt, session):
+   # Will overwrite the submission method setup in dbt_project.yml
+   dbt.config(submission_method="bigframes")
+
+   bdf = dbt.ref("my_sql_model") #loading from prev step
+   <process bigquery dataframe >
+   return bdf
+```
+
+```python
+# Already set bigframes submission method in dbt_project.yml
+def model(dbt, session):  
+   data = {"foo": [1, 2], "bar": [3, 4]}
+   return bpd.DataFrame(data=data)
+```
+
+2. **Execute the dbt pipeline**
```

Comment:
To keep this short and sweet - let's just have it execute the example dbt model specifically. This isn't a dbt 101